### PR TITLE
Add missing enum & ctor for PackedVector4Array implementation

### PR DESCRIPTION
**Pull request by Naros**
@dsnopek I think this may have been overlooked in https://github.com/godotengine/godot-cpp/pull/1456

### CODE REVIEW

File: include/godot_cpp/variant/variant.hpp [hunk 1]
```diff
@@ -211,6 +212,7 @@ class Variant {
 	Variant(const PackedStringArray &v);
 	Variant(const PackedVector2Array &v);
 	Variant(const PackedVector3Array &v);
+	Variant(const PackedVector4Array &v);
```
Comment (akien-mga) [inline include/godot_cpp/variant/variant.hpp:215]: This should be after PackedColorArray, to stay consistent with the inconsistent order of `Variant::Type`. Same below.

**Comment by dsnopek** [comment]
Thanks!

However, this only appears to update `variant.hpp` - shouldn't the implementations for these methods need to be added to `variant.cpp` as well?

**Comment by Naros** [comment]
> Thanks!
> 
> However, this only appears to update `variant.hpp` - shouldn't the implementations for these methods need to be added to `variant.cpp` as well?

Ya it does, fixing the PR.

**Review by dsnopek — APPROVED** [review]
Thanks!

I haven't tested it, but the code looks good to me. Assuming this passes CI, I think it's good to go :-)

**Pull request merged**